trade-finance-manager-api/src/v1/api.js

```diff
@@ -1204,15 +1204,15 @@ const getUtilisationReportsReconciliationSummary = async (submissionMonth) => {
 };
 
 /**
- * @param {string} _id
- * @returns {Promise<import('../types/utilisation-reports').UtilisationReportResponseBody>}
+ * @param {string} id
+ * @returns {Promise<import('@ukef/dtfs2-common/').UtilisationReportResponseBody>}
  */
-const getUtilisationReportById = async (_id) => {
-  if (!isValidMongoId(_id)) {
-    throw new Error(`Invalid MongoDB _id provided: '${_id}'`);
+const getUtilisationReportById = async (id) => {
+  if (!Number.isInteger(id)) {
```

Comment:
As mentioned above, might be nice to wrap the `isInteger` check in an `isValidSqlId` helper :)